hal/src/boron/network/sara_ncp_client.cpp

```diff
@@ -268,10 +268,16 @@ int SaraNcpClient::initParser(Stream* stream) {
         // Cellular Global Identity (partial)
         if (r >= 3) {
             auto rat = r >= 4 ? static_cast<CellularAccessTechnology>(val[3]) : self->act_;
+            if (self->ncpId() == PLATFORM_NCP_SARA_R410) {
+                if (rat == CellularAccessTechnology::LTE) {
+                    rat = CellularAccessTechnology::LTE_CAT_M1;
+                    //LOG(INFO, "RAT changed to report Cat-M1");
```

Comment:
Do we need to keep these commented out comments here and in all the other places as well?